makemkv.js

```diff
@@ -213,7 +213,7 @@ class MakeMkv {
             trackIds.map((trackId) => this.ripQueue.driveId.add(trackId));
         }
 
-        if (!this.ripQueue.driveId.length) {
+        if (!this.ripQueue.driveId.size) {
```

Comment:
Kind of confused. `driveId` is a Set, but you're interacting with it like it's an array. Not sure which you meant for it to be, so I stuck with a set.

Reply:
Yep this was wrong. Looks like the conditional doesn't do anything, so not really sure what I was going for honestly